soaktest/README.md

```diff
@@ -0,0 +1,39 @@
+# Information for soak test for go client
+The soak test contains 2 applications:
+soakclient.go for one application : Create Producer-1 to keep sending messages to the input-topic.
+Create Consumer-2 to receive messages from the output-topic.
+
+soakclient_transaction.go for the other application: Create Consumer-1 to receive messages from the input-topic.
+Create transactional producer Producer-2 to sending the received messages to the output-topic.
+The initTransaction API registers a transactional.id when creating the producer.
+The producer sends the messages to the output-topic.
+Commit transaction after sending 100 messages each time or 1 second.
+
+# Instruction on configuration and running
+
+# Build soaktest API
+    $ cd confluent-kafka-go/soaktest
```

Comment:
nit: people are probably reading this from the confluent-kafka-go topdir, so remove the confluent-kafka-go/ prefix here and below.